src/modules/integrator-guide/pages/compose-configuration/index.adoc

```diff
@@ -867,40 +867,75 @@ length(title) < 5
 
 == Page Block Refresh
 
-Page blocks have a refresh option to automatically renew their data without having to reload the entire page. 
-
-This is useful for page blocks that display data that changes frequently, such as a list of new leads, an update to a chart, or a list of recent comments.
+Page blocks have a refresh option to automatically renew their data without having to reload the entire page, this is useful for page blocks that display data that changes frequently, such as a list of new leads, an update to a chart, or a list of recent comments.
 
 [IMPORTANT]
 ====
-Automation, File, Content, Nylas, Navigation, and Geometry blocks do not have the refresh option.
+Automation, file, content, nylas, navigation, and geometry blocks do not have the refresh option.
 ====
 
-
 === Enabling Auto-refresh
 
 When set, page blocks that have this configuration will automatically refresh based on a set interval.
 
-To activate auto refresh, navigate to the page builder, choose a block that supports refresh, and go to its general tab, there you will find the refresh option.
+To activate auto-refresh, navigate to the page builder, choose a block that supports refresh, and go to its general tab. There you will find the refresh option.
 
-You'll want to enter a numerical value into the input above the *Show refresh button* toggle, click save and close, save the page and you should see the block start to every time your interval elapses.
+You'll want to enter a value expressed in seconds into the input above the *Show refresh button* toggle, click save and close, save the page and you should see the block start to refresh when your interval elapses.
 
 The lowest interval allowed is 5 seconds. 
-Setting an interval lower than that would make the input default to 5 seconds. 
 To turn auto-refresh off on a block, you need to set its value to 0.
 
-image:compose-configuration/page-block-general.png["Page block general tab", role="data-zoomable"]
+.Page Block General Tab
+[annotation,role="data-zoomable"]
+----
+{
+  "image": {
+    "rel": "compose-configuration/page-block-general.png",
```

Comment:
I feel like we can do a bit more with this image here.
What I'd consider doing is moving it up to where you outline how to get to this configuration.
I'd also annotate the section where you configure the refresh.

So...

```
...How do you get there...

Image

...
```